rfc/0060-mod-menu.md

```diff
@@ -0,0 +1,76 @@
+
+## Summary
+
+This document outlines the process for creating an official Mod Menu for usage with Quilt alongside an accompanying team that would maintain it. This mod menu will act as a way to view loaded mods, change configs, read credits, and get links to contacts from within Minecraft.
+
+
+## Motivation
+   As of right now, we do not supply an official Mod Menu for Quilt. The current solution is TerraformerMC's Mod Menu for Fabric, and that does not fully support our additions (Full Quilt support, Quilt Config, quilt.mod.json spec improvements, etc.). Having a standardized Mod Menu would allow for support for these features. Furthermore, it would also be an officially endorsed mod, which would help us lead players to it, and we at The Quilt Project would be able to ensure its quality.
+
+
+## Explanation
+
+### Integration
+A toggle in the Quilt Installer will allow for this mod to be installed at the same time as loader
```

Comment:
How will updates be sent out? This feature of mod downloads at installation needs to be expanded in my opinion.

Reply:
Updating is way out of scope for this and shouldn't be necessary.

Reply:
After discussing with Finn and Cheater, we decided the integration should probably be its own RFC. I'll be removing it from this one